protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -5009,3 +5016,38 @@ message AMMEdge {
   // Cursor that can be used to fetch further pages.
   string cursor = 2;
 }
+
+message EstimateAMMBoundsRequest {
+  // Base price of the AMM pool, the price is an integer, for example `123456` is a correctly
+  // formatted price of `1.23456` assuming market configured to 5 decimal places.
+  string base_price = 1;
+  // Upper price of the AMM pool, the price is an integer, for example `123456` is a correctly
+  // formatted price of `1.23456` assuming market configured to 5 decimal places.
+  string upper_price = 2;
```

Comment:
This and `lower_price` should be optional since you can have a AMM that is only one sided i.e can only go long, and can only go short.

Reply:
done